lighthouse-core/formatters/partials/table.html

```diff
@@ -0,0 +1,68 @@
+<style>
+.table_list {
+  margin-top: 8px;
+  border: 1px solid #EBEBEB;
```

Comment:
There's a PR that alphabetizing selectors was brought up (can't find it now), but we deemed it unrealistic without tooling.